.travis.yml

```diff
@@ -6,13 +6,13 @@ os:
 - linux
 dist:
 # take the latest supported LTS version
-- trusty
+- xenial
 
 # install dependencies
 
 before_install:
 # install latest r
-- sudo sh -c 'echo "deb http://cran.rstudio.com/bin/linux/ubuntu trusty/" >> /etc/apt/sources.list'
+- sudo sh -c 'echo "deb https://cloud.r-project.org/bin/linux/ubuntu xenial/" >> /etc/apt/sources.list'
```

Comment:
same here (?)
you could use: https://cloud.r-project.org/bin/linux/ubuntu  bionic-cran35